src/core/src/lib/extensions/core/core.ts

```diff
@@ -17,6 +17,8 @@ import { Subject } from 'rxjs';
 
 /** @experimental */
 export class CoreExtension implements FormlyExtension {
+  priority = -1;
```

Comment:
instead I would prefer passing priority through `NgModule` config:
```ts
extensions: [
  { name: 'core', extension: core, priority: ... },
  ...
],
```

Reply:
Okay good point, I will change it to work like that.